cypress/e2e/latest_transaction_list_Xchain.cy.ts

```diff
@@ -0,0 +1,348 @@
+        cy.get(':nth-child(1) > .MuiPaper-root > .MuiCardContent-root > .MuiBox-root > .MuiTypography-h4').invoke("text")
+        .then(textV => {
+            var numberV =parseInt(textV)
+            if(numberV == validators.value.length){
+                cy.log('success')
+            }else{
+                cy.log('failed number validators')
+            }
+        })
+        })
+        cy.wait('@txcount').then(()=>{
+            cy.get(':nth-child(2) > .MuiPaper-root > .MuiCardContent-root').should('be.visible')
+            cy.get(':nth-child(2) > .MuiPaper-root > .MuiCardContent-root > .MuiBox-root > .MuiTypography-root').invoke('text')
+            .then(textCount =>{
+                var txC =parseInt(textCount)
+                if(txC == txcount.aggregates.transactionCount){
+                    cy.log('success tx count')
+                }else{
+                    cy.log('failed tx count')
+                }
+            })
+        })
+        cy.wait('@transactions').then(()=>{
+            cy.get('.css-1voxykv-MuiPaper-root').should('be.visible')
+            cy.get(':nth-child(1) > .MuiGrid-grid-md-4 > .MuiGrid-container > .MuiGrid-grid-xs-8 > a > .MuiTypography-root').invoke('text')
+            .then(textId =>{
+                if(textId == transactionsBody.transactions[0].id){
+                    cy.log('success txId')
+                }else{
+                    cy.log('failed txId')
+                }
+            })
+            cy.get('.MuiGrid-grid-xs-4 > .MuiChip-root > .MuiChip-label').invoke('text')
+            .then(textType =>{
+                if(textType == transactionsBody.transactions[0].type){
+                    cy.log('success type')
+                }else{
+                    cy.log('failed type')
+                }
+            })
+            cy.get(':nth-child(1) > .MuiGrid-grid-md-8 > .MuiGrid-grid-lg-8 > :nth-child(1) > [style="width: 100%;"] > .MuiGrid-grid-xs-9 > .MuiGrid-container > .MuiGrid-grid-xl-7 > a > .MuiTypography-root').invoke('text')
+            .then(textFrom =>{
+                if(textFrom == transactionsBody.transactions[0].inputs[0].output.addresses){
+                    cy.log('success address from')
+                }else{
+                    cy.log('failed address from')
+                }
+            })
+            cy.get(':nth-child(1) > .MuiGrid-grid-md-8 > .MuiGrid-grid-lg-8 > :nth-child(2) > [style="width: 100%;"] > .MuiGrid-grid-xs-9 > :nth-child(1) > .MuiGrid-grid-xl-7 > a > .MuiTypography-root').invoke('text')
+            .then(textTo =>{
+                if(textTo == transactionsBody.transactions[0].outputs[0].addresses){
+                    cy.log('success adress to')
+                }else{
+                    cy.log('failde address to')
+                }
+            })
+            cy.get(':nth-child(1) > [style="width: 100%;"] > .MuiGrid-grid-xs-9 > .MuiGrid-container > .MuiGrid-grid-xl-5 > .MuiBox-root > .MuiTypography-subtitle1').invoke('text')
```

Comment:
get an error message, as below :
```
Timed out retrying after 4000ms: Expected to find element: :nth-child(1) > [style="width: 100%;"] > .MuiGrid-grid-xs-9 > .MuiGrid-container > .MuiGrid-grid-xl-5 > .MuiBox-root > .MuiTypography-subtitle1, but never found it.
```
In [cypress/e2e/latest_transaction_list_Xchain.cy.ts:316:16](http://localhost:5001/__/#)